homeassistant/components/cmus/media_player.py

```diff
@@ -153,30 +154,6 @@ def set_volume_level(self, volume: float) -> None:
         """Set volume level, range 0..1."""
         self._remote.cmus.set_volume(int(volume * 100))
 
-    def volume_up(self) -> None:
-        """Set the volume up."""
-        left = self.status["set"].get("vol_left")
-        right = self.status["set"].get("vol_right")
-        if left != right:
-            current_volume = float(left + right) / 2
-        else:
```

Comment:
I'm not sure how crucial this calculation of an average volume is for the volume up / down.

Reply:
The averaging is already done in the state update for `_attr_volume_level` so the base implementation of volume up/down will have the same behavior.